appintro/src/main/res/values-sw600dp/dimen-sw600dp.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <dimen name="appintro_statusbar_height">24dp</dimen>
+
+    <dimen name="appintro_bottombar_height">70dp</dimen>
+    <dimen name="appintro2_bottombar_height">90dp</dimen>
+    <dimen name="appintro2_fab_size">100dp</dimen>
```

Comment:
I'm unsure this is achieving what you want. I've done some testing locally and the FAB size is not affected because the background drawable is center cropped.

Reply:
```suggestion
    <dimen name="appintro2_fab_size">98dp</dimen>
```

Please change to this. We should account for `1dp` margin on every boarder.